src/core/evaluator.js

```diff
@@ -723,29 +723,70 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           return errorFont();
         }
       }
+      let fallbackFont = false;
+      let fallbackFontProperties = false;
       if (!fontRef) {
-        warn('fontRef not available');
-        return errorFont();
+        // warn('fontRef not available');
+        // return errorFont();
+
+        // fallback to some default font
+        let widthsWithNulls = [null, null, null, null, null, null, null,
```

Comment:
For the general PDF.js library, you should use widths corresponding a *standard font* and we also want to avoid defining this data inline.

Please find already embedded *standard font* widths in https://github.com/mozilla/pdf.js/blob/master/src/core/metrics.js, where the `Helvetica` ones seem like a good choice for the fallback font being constructed here.